docs/dev/apis/lethallib/custom-enemies/unity.md

```diff
@@ -0,0 +1,160 @@
+---
+prev: true
+next: true
+description: A tutorial on how to add custom enemies by using LethalLib.
+---
+# Unity
+
+You can open the Unity project included in the repository by choosing to open a project from disk, and selecting the `UnityProject` folder. When Unity has loaded the project, look into the ToiletLeech folder for the assets that make up the asset bundle.
```

Comment:
Yeah I also want to do this, but I want a different repository for a dedicated example enemy (even just by renaming Toilet Leech to ExampleEnemy), as toilet leech has just basically been me getting the enemy working in the first place. I guess I could make the repo on my account, but also it would make a lot of sense for it to be by the lethal company community organization as I stated in my original message, as the guide will be on the wiki.

Reply:
Done! I also made an LC-ExampleEnemy repository.